packages/broker/package.json

```diff
@@ -63,7 +63,7 @@
     "qs": "^6.10.1",
     "split2": "^3.2.2",
     "streamr-client": "^5.4.2",
-    "streamr-network": "^24.3.3",
+    "streamr-network": "^24.3.4-alpha.0",
```

Comment:
Yeah I think we can just leave the alphas versions off, doesn't help a lot and creates a bunch of busywork. Point at whatever version is being used in the local repo.